flink-python/src/test/java/org/apache/flink/client/cli/PythonProgramOptionsTest.java

```diff
@@ -81,7 +92,7 @@ public void testCreateProgramOptionsWithPythonCommandLine() throws CliArgsExcept
 	}
 
 	@Test
-	public void testCreateProgramOptionsWithLongOptions() throws CliArgsException {
+	public void testCreateProgramOzptionsWithLongOptions() throws CliArgsException {
```

Comment:
Why do we change the test name?

Reply:
Sorry for changing the irrelevant test case accidentally.